src/main/java/com/epam/test_generator/services/SuitService.java

```diff
@@ -61,4 +66,37 @@ public void removeSuit(long suitId) {
 
         caseVersionDAO.delete(suit.getCases());
     }
+
+    public void updateSuitRowNumber(List<SuitRowNumberUpdateDTO> rowNumberUpdates) {
+        if (rowNumberUpdates.isEmpty()) {
+            throw new BadRequestException("The list has not to be empty");
+        }
+
+        for (SuitRowNumberUpdateDTO s : rowNumberUpdates) {
+            if (s.getId() == null) {
+                throw new BadRequestException("Id has not to be null");
+            }
```

Comment:
It's check in SuitRowNumberUpdateDTO with NotNull annotation, so this check is unnecessary here.